tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+      "outDir": "./esm",
+      "baseUrl": "./src",
+      "target": "es6",
+      "module": "esnext",
+      "lib": [
+        "esnext"
+      ],
+      "declaration": true,
+      "allowJs": true,
+      "skipLibCheck": true,
+      "esModuleInterop": true,
```

Comment:
similar question with the config here - was this trial-and-error to get our existing code to work with typescript, or a set of options you've used before and like, or something?

Reply:
Copied from Parser-JS 2.0 development https://github.com/asyncapi/parser-js/blob/next-major/tsconfig.json